docs/design/jet/023-kafka-connect-source-connectors.md

```diff
@@ -189,6 +189,16 @@ There are also nice to have features:
 - Add benchmarks for Kafka connect source
 - Add Soak tests for Kafka connect source
 
+### Task Distribution
+**TaskMaxProcessorMetaSupplier**  evenly distributes the number of tasks specified in the properties input to cluster members.    
+It creates **TaskMaxProcessorSupplier** objects and assigns them a processor order number.   
+The first processor with **processor order 0** is the **master** processor    
+The **master processor** distributes task assignments to all other processors via a 
+**Reliable Topic** named **"__jet." + executionId**  
```

Comment:
I'd add some context info to the name, e.g. `kafka-connect-sync` or something like that, so users who sees it have some context and won't be scared by long numbers ;)